documentation/asciidoc/stories/assembly_configuring_client_certificates.adoc

```diff
@@ -7,6 +7,17 @@ ifdef::context[:parent-context: {context}]
 Add client trust stores to your project and configure {brandname} to allow connections only from clients that present valid certificates.
 This increases security of your deployment by ensuring that clients are trusted by a public certificate authority (CA).
 
+[NOTE]
```

Comment:
I think this has been added to in the wrong section of the documentation. The issue is related to xsite connections, but you have added the NOTE to a section detailing how to configure client-certificate authentication (applicable to local cluster access, not xsite).

I think you need  to be adding the note to this topic:

documentation/asciidoc/topics/proc_configuring_sites_automatically.adoc

Reply:
Thanks, @ryanemerson . I moved the note to the section that you referenced.